src/standalone/standard_attached.rs

```diff
@@ -21,7 +21,7 @@ impl StandaloneAttachedStandardClient {
     }
 }
 
-#[uniffi::export(async_runtime = "tokio")]
+#[uniffi::export]
```

Comment:
I'm a bit surprised this works. I assume it's because we don't actually do anything to do with tokio inside any of these functions. Our test coverage should keep us from breaking this accidentially IIRC.

Reply:
Yeah, standalone doesn't use tokio so we don't have the annotation on that half. I assume that means that it doesn't matter, but just consistency